nototools/create_image.py

```diff
@@ -31,6 +31,11 @@
 import pango
 import pangocairo
 
+try:
+  basestring
+except NameError:
+  basestring = str
```

Comment:
yeah, let's add basestring symbol to nototools.py23 since it's used more than once

Reply:
That's what I did originally, but it doesn't work this way in Python 2 to say 
`from nototools.py23 import basestring` 
so need to change it to  
```
try:
    basestring = basestring
except NameError:
    basestring = str

```